cmd/kubeadm/app/cmd/completion.go

```diff
@@ -260,7 +260,7 @@ autoload -U +X bashcompinit && bashcompinit
 # use word boundary patterns for BSD or GNU sed
 LWORD='[[:<:]]'
 RWORD='[[:>:]]'
-if sed --help 2>&1 | grep -q GNU; then
+if sed --help 2>&1 | grep -q "GNU\|BusyBox"; then
```

Comment:
I would suggest a bit different approach here. Busybox and Toybox contain (almost) the same homegrown version of sed that claims to be somewhat GNU compatible. Instead of listing all possibilities here, we can simply do:
```shell
if sed --version 2>&1 | grep -q GNU; then
```

GNU sed would dump its version and license, Busybox/Toybox sed would state that it's not GNU sed version 4.0, and BSD sed would obviously return an error.

Reply:
^ @wangrzneu please address the above and keep the commits squashed to 1.
thanks.

/hold